### PR TITLE
Bump to Instant Client 19.11

### PR DESCRIPTION
**Pull request by cjbj**
Bump to Instant Client 19.11.

Mention the package registry.

### CODE REVIEW

File: OracleInstantClient/README.md [hunk 1]
```diff
@@ -29,6 +29,20 @@ Oracle Database 12.1 or later.  Applications using Oracle Call Interface (OCI)
 19, 18 or 12.2 can connect to Oracle Database 11.2 or later.  Some tools may
 have other restrictions.
 
+## Images
+
+Pre-built images for Instant Client are in the [GitHub Package
```
Comment (Djelibeybi) [inline OracleInstantClient/README.md:34]: It should be GitHub _Container_ Registry, i.e. `ghcr.io`. :)

File: OracleInstantClient/README.md [hunk 2]
```diff
@@ -29,6 +29,20 @@ Oracle Database 12.1 or later.  Applications using Oracle Call Interface (OCI)
...
+Registry](https://github.com/orgs/oracle/packages).  They are built from the
+the Dockerfiles in this repository.
+
+For example, to pull an Oracle Linux 8 image with Instant Client 21c, execute:
```
Comment (Djelibeybi) [inline OracleInstantClient/README.md:38]: This is probably better phrased as, "_to pull an  image based on Oracle Linux 8 that already has the Oracle Instant Client 21c installed_" and I would recommend calling out whether or not its based on `oraclelinux:8` which means `dnf` is available or `oraclelinux:8-slim` which only has `microdnf`.

File: OracleInstantClient/README.md (hunk 1 above)
Comment (Djelibeybi) [inline OracleInstantClient/README.md:34]: It should be GitHub _Container_ Registry, i.e. `ghcr.io`. :)

File: OracleInstantClient/README.md [hunk 3]
```diff
@@ -29,6 +29,20 @@ Oracle Database 12.1 or later.  Applications using Oracle Call Interface (OCI)
...
+
+```
+docker pull ghcr.io/oracle/oraclelinux8-instantclient:21
+```
+
+Images that include various languages are also available.
+
 ## Building the Oracle Instant Client 21 Images
```
Comment (Djelibeybi) [inline OracleInstantClient/README.md:46]: Should this be "Building an Oracle Instant Client image"?

**Review by Djelibeybi — CHANGES_REQUESTED** [review]

File: OracleInstantClient/README.md (hunk 2 above)
Comment (cjbj) [reply OracleInstantClient/README.md:38]: I'll do some tweaks. But your suggestion is wordy. And the slim distinction is overkill for this brief reference.

File: OracleInstantClient/README.md (hunk 3 above)
Comment (cjbj) [reply OracleInstantClient/README.md:46]: This has the same format as all the other version headings.

**Review by Djelibeybi — APPROVED** [review]

**Comment by cjbj** [comment]
@Djelibeybi thank you

**Pull request merged**